### ISSUE TITLE
command line tool to simulate a server

### ISSUE THREAD

**Issue by ekondrashev** [yegor256/rultor-remote#17]
Let's create a command line tool, which should simulate the server. I should be able to run it locally:

`$ rultor deploy`
It should understand its location, and fetch .rultor.yml correctly.

This issue was moved from [rultor#549](https://github.com/yegor256/rultor/issues/549)

**Comment by alex-palevsky** [comment]
@yegor256 please pay attention to this issue ([par.21](http://at.teamed.io/policy.html#21))

**Comment by alex-palevsky** [comment]
@ekondrashev thanks for the ticket, your account was topped for 15 mins, payment 57726e15499c8d1085000727

**Comment by tannakartikey** [comment]
@yegor256 I am configuring Rultor to use custom container. I have setup automated build on Docker Hub.
So every time I encounter an error if I push it to Docker Hub, build takes almost an hour before I could ask Rultor to try again.
`rultor deploy` would help a lot especially while debugging/configuring.
Are we going to have it anytime in the near future?

**Comment by tannakartikey** [comment]
@yegor256 Sure. Below is how I have understood the application of `rultor deploy`. Please let me know if it is correct.
When `rultor deploy` is given, `rultor-remote` will fetch `.rultor.yml` correctly and start the container as per the `deploy` script configured in `.rultor.yml` - locally.
It can also support `release` command, right?
I think `merge` should not be supported by `rultor-remote` as we need to run it on Github PR to make it function properly.

**Comment by yegor256** [comment]
@tannakartikey yes, you got it right. We will support only `deploy` and `release`. Please, try to implement it. I'm very interested to see it working. I will use it too.
